libraries/mongodb.rb

```diff
@@ -80,17 +87,11 @@ def self.configure_replicaset(node, name, members)
       "Configuring replicaset with config: #{rs_members}"
     )
 
-    rs_member_ips = []
-    members.each_index do |n|
-      port = members[n]['mongodb']['config']['port']
-      rs_member_ips << { '_id' => n, 'host' => "#{members[n]['ipaddress']}:#{port}" }
-    end
-
     admin = connection['admin']
     cmd = BSON::OrderedHash.new
     cmd['replSetInitiate'] = {
       '_id' => name,
-      'members' => node['mongodb']['use_ip_address'] ? rs_member_ips : rs_members,
+      'members' => rs_members,
```

Comment:
I'm actually not killing it :P
I moved this part of the code to the [extracted function](https://github.com/mateusduboli/mongodb/blob/f40f24c888bc84fbd455e0e91291821119e22b05/libraries/mongodb.rb#L29), I want to do a little more but would rather split in different PRs.

I also used the same variable to set ports, since on the old code there were 2 variables for it.